src/transformers/models/auto/image_processing_auto.py

```diff
@@ -62,6 +62,7 @@
         ("dpt", "DPTImageProcessor"),
         ("efficientformer", "EfficientFormerImageProcessor"),
         ("efficientnet", "EfficientNetImageProcessor"),
+        ("fastvit", "ViTImageProcessor"),
```

Comment:
I am using ViTImageProcessor to avoid repeat the same code, the unique difference is the image size, by default in ViTImageProcessor is 224 and in FastViT is 256, Do you think I should add a "FastViTImageProcessor"? It won't take much time but i preferred to keep it this way to have the code as clean as possible

Reply:
That's a good question.
@ArthurZucker, as you're have more experience than me, what do you think it's better to be done here?